dm/worker/subtask.go

```diff
@@ -171,6 +171,7 @@ func (st *SubTask) Init() error {
 			return terror.Annotatef(err, "fail to get fresh status of subtask %s %s", st.cfg.Name, u.Type())
 		} else if !isFresh {
 			skipIdx = i
+			// TODO: why after print it, nothing continued?
 			st.l.Info("continue unit", zap.Stringer("unit", u.Type()))
```

Comment:
? We iterate from sync unit to dump unit, if sync unit is not flesh, continue sync unit.

Reply:
it's a bug that I forget to reset `st.ctx` in `resetAllSubTasks` on previous commits, at that time I see "continue unit" but no unit running because of a done context. Will remove this TODO